doc/integrator/functionality.rst

```diff
@@ -4,20 +4,16 @@ Functionalities
 c2cgeoportal provides the concept of *functionality* that can be used to customize
 the application according to the user's permissions.
 
-A functionality may be associated to users through different ways:
+A functionality may be associated to users through the admin interface.
 
-1. Functionalities for anonymous users are defined through the
-   ``functionalities:anonymous`` variable in the ``vars.yaml`` configuration file.
-2. Functionalities for authenticated users are defined through the
-   ``functionalities:registered`` variable in the ``vars.yaml`` configuration file.
-3. Functionalities for specific roles are defined in the database through the administration interface.
-4. Functionalities for specific users are defined in the database through the administration interface.
+There are some technical user names `anonymous`, `registred` and `intranet`.
 
-Each level overrides the previous ones in the order indicated above.
-For example, if the user is authenticated and has associated functionalities in
-the ``user`` database table, then the ``functionalities:anonymous`` and
-``functionalities:registered`` configuration variables, as well as any
-functionality associated with his/her role, will be ignored.
+The functionalities resolution follows the following priorities::
+
+    - from current user for logged users,
+    - from `registred` user for logged users,
```

Comment:
"registered"
But, more important: what is the difference between this line and the previous line? Both are "for logged users"

Reply:
The difference is that's not the same user...

Reply:
Proposition for the form, I make some suppositions on the content:

User get functionnalities from, by priority order: 
If authenticated :
- roles associated to his user profile
- role named "registered"
If his IP is included in the intranet networks:
- role named "intranet"
In all cases:
- role named "anonymous"

Reply:
What do you think @jwkaltz I'm not sure that's clearer...

Reply:
I like @arnaud-morvan 's suggestion, it seems clearer to me